recipes/tttrlib/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "tttrlib" %}
+{% set version = "0.25.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Fluorescence-Tools/tttrlib/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e15604ef063257cd87d3bdf266c1d5a448189508f51cdc4156a93cb2b46fe807
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - llvm-openmp        # [osx]
+    - libgomp            # [linux]
+    - pkg-config         # [not win]
+    - cmake
+    - ninja
+    - make               # [linux]
+    - doxygen
+    - swig 4.2.*
+    - python
+    - numpy
```

Comment:
```suggestion
```
Typically, we use python from host unless cross-compiling. See the conda-forge docs section about cross-compilation.